### ISSUE TITLE
Failing to build on Hyprland 0.45.2-3

### ISSUE THREAD

**Issue by smtucker** [horriblename/hyprgrass#180]
Does not build when installing with hyprpm

**Log**

```
❯ hyprpm update -f -v --no-shallow                              
! Using --force, I hope you know what you are doing.
[v] version returned: Hyprland 0.45.2 built from branch  at commit 12f9a0d0b93f691d4d9923716557154d74777b0a  ([gha] Nix: update inputs).
Date: Tue Nov 19 21:47:18 2024
Tag: v0.45.2, commits: 5451
built against aquamarine 0.5.0


flags set:
debug


[v] parsed commit 12f9a0d0b93f691d4d9923716557154d74777b0a at branch  on Tue Nov 19 21:47:18 2024, commits 5451

! Cloning https://github.com/hyprwm/Hyprland, this might take a moment.                                                                                                                                        
✔ Hyprland cloned                                                                                                                                                                                              
[v] will run: cd /tmp/hyprpm/hyprland-shelby && git checkout 12f9a0d0b93f691d4d9923716557154d74777b0a 2>&1                                                                                                     
[v] git returned (co): Note: switching to '12f9a0d0b93f691d4d9923716557154d74777b0a'.                                                                                                                          

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 12f9a0d0 [gha] Nix: update inputs

[v] git returned (rs): rm 'subprojects/tracy'                                                                                                                                                                  
HEAD is now at 12f9a0d0 [gha] Nix: update inputs

✔ checked out to running ver                                                                                                                                                                                   
! configuring Hyprland                                                                                                                                                                                         
[v] setting PREFIX for cmake to /home/shelby/.local/share/hyprpm/headersRoot                                                                                                                                   
[v] cmake returned: Not searching for unused variables given on the command line.                                                                                                                              
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Gathering git info
-- Found PkgConfig: /usr/bin/pkg-config (found version "2.1.1")
-- Checking for module 'udis86>=1.7.2'
--   Package 'udis86', required by 'virtual:world', not found
-- Found Python3: /usr/bin/python3.12 (found version "3.12.7") found components: Interpreter
Building itab.c/itab.h...
-- udis86 dependency not found, falling back to subproject
-- Found wayland-protocols at //usr/share/wayland-protocols
-- Found wayland-scanner pkgdatadir at //usr/share/wayland
-- Configuring Hyprland in Release with CMake
-- Checking deps...
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found OpenGL: /usr/lib/libOpenGL.so  found components: GLES3
-- Checking for module 'hyprutils>=0.2.1'
--   Found hyprutils, version 0.2.6
-- Checking for module 'aquamarine>=0.4.2'
--   Found aquamarine, version 0.5.0
-- Checking for modules 'xkbcommon;uuid;wayland-server>=1.22.90;wayland-protocols;cairo;pango;pangocairo;pixman-1;xcursor;libdrm;libinput;gbm;gio-2.0;hyprlang>=0.3.2;hyprcursor>=0.1.7;hyprutils>=0.2.3'
--   Found xkbcommon, version 1.7.0
--   Found uuid, version 2.40.2
--   Found wayland-server, version 1.23.1
--   Found wayland-protocols, version 1.38
--   Found cairo, version 1.18.2
--   Found pango, version 1.54.0
--   Found pangocairo, version 1.54.0
--   Found pixman-1, version 0.44.0
--   Found xcursor, version 1.2.3
--   Found libdrm, version 2.4.123
--   Found libinput, version 1.27.0
--   Found gbm, version 24.2.7-arch1.1
--   Found gio-2.0, version 2.82.2
--   Found hyprlang, version 0.5.3
--   Found hyprcursor, version 0.1.10
--   Found hyprutils, version 0.2.6
-- Looking for execinfo.h
-- Looking for execinfo.h - found
-- Configuration supports execinfo
-- Looking for backtrace in execinfo
-- Looking for backtrace in execinfo - not found
-- Looking for sys/timerfd.h
-- Looking for sys/timerfd.h - found
-- Checking for module 'epoll-shim'
--   Package 'epoll-shim', required by 'virtual:world', not found
-- XWAYLAND Enabled (NO_XWAYLAND not defined) checking deps...
-- Checking for modules 'xcb;xcb-render;xcb-xfixes;xcb-icccm;xcb-composite;xcb-res;xcb-errors'
--   Found xcb, version 1.17.0
--   Found xcb-render, version 1.17.0
--   Found xcb-xfixes, version 1.17.0
--   Found xcb-icccm, version 0.4.2
--   Found xcb-composite, version 1.17.0
--   Found xcb-res, version 1.17.0
--   Found xcb-errors, version 1.0.1
-- SYSTEMD support is requested (NO_SYSTEMD not defined)...
-- Setting precompiled headers
-- Setting link libraries
-- Checking for module 'hyprland-protocols>=0.4.0'
--   Package 'hyprland-protocols', required by 'virtual:world', not found
-- hyprland-protocols subproject set to subprojects/hyprland-protocols
-- Checking for module 'hyprutils>=0.1.1'
--   Found hyprutils, version 0.2.6
-- Checking for modules 'tomlplusplus;hyprutils>=0.1.1'
--   Found tomlplusplus, version 3.4.0
--   Found hyprutils, version 0.2.6
-- Configuring done (2.6s)
-- Generating done (0.0s)
-- Build files have been written to: /tmp/hyprpm/hyprland-shelby/build

✔ configured Hyprland                                                                                                                                                                                          
[v] installation will run: sed -i -e "s#PREFIX = /usr/local#PREFIX = /home/shelby/.local/share/hyprpm/headersRoot#" /tmp/hyprpm/hyprland-shelby/Makefile && cd /tmp/hyprpm/hyprland-shelby && make installheaders
 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╍━━━━━━━━━  4 / 5  Installing sources[v] installer returned: # remove previous headers from hyprpm's dir                                                              
rm -fr /home/shelby/.local/share/hyprpm/headersRoot/include/hyprland
mkdir -p /home/shelby/.local/share/hyprpm/headersRoot/include/hyprland
mkdir -p /home/shelby/.local/share/hyprpm/headersRoot/include/hyprland/protocols
mkdir -p /home/shelby/.local/share/hyprpm/headersRoot/share/pkgconfig
cmake --build ./build --config Release --target generate-protocol-headers
[0/2] Re-checking globbed directories...
[1/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/hyprland-global-shortcuts-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/hyprland-global-shortcuts-v1.hpp
[2/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/text-input-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/text-input-unstable-v1.hpp
[3/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/hyprland-toplevel-export-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/hyprland-toplevel-export-v1.hpp
[4/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/wlr-screencopy-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/wlr-screencopy-unstable-v1.hpp
[5/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/wlr-gamma-control-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/wlr-gamma-control-unstable-v1.hpp
[6/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/wlr-foreign-toplevel-management-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/wlr-foreign-toplevel-management-unstable-v1.hpp
[7/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/wlr-output-power-management-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/wlr-output-power-management-unstable-v1.hpp
[8/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/virtual-keyboard-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/virtual-keyboard-unstable-v1.hpp
[9/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/wlr-virtual-pointer-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/wlr-virtual-pointer-unstable-v1.hpp
[10/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/input-method-unstable-v2.cpp, /tmp/hyprpm/hyprland-shelby/protocols/input-method-unstable-v2.hpp
[11/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/wlr-output-management-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/wlr-output-management-unstable-v1.hpp
[12/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/kde-server-decoration.cpp, /tmp/hyprpm/hyprland-shelby/protocols/kde-server-decoration.hpp
[13/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/wlr-data-control-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/wlr-data-control-unstable-v1.hpp
[14/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/hyprland-focus-grab-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/hyprland-focus-grab-v1.hpp
[15/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/wlr-layer-shell-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/wlr-layer-shell-unstable-v1.hpp
[16/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/hyprland-ctm-control-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/hyprland-ctm-control-v1.hpp
[17/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/wayland-drm.cpp, /tmp/hyprpm/hyprland-shelby/protocols/wayland-drm.hpp
[18/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/tearing-control-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/tearing-control-v1.hpp
[19/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/fractional-scale-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/fractional-scale-v1.hpp
[20/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/xdg-output-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/xdg-output-unstable-v1.hpp
[21/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/cursor-shape-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/cursor-shape-v1.hpp
[22/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/idle-inhibit-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/idle-inhibit-unstable-v1.hpp
[23/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/relative-pointer-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/relative-pointer-unstable-v1.hpp
[24/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/xdg-decoration-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/xdg-decoration-unstable-v1.hpp
[25/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/alpha-modifier-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/alpha-modifier-v1.hpp
[26/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/ext-foreign-toplevel-list-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/ext-foreign-toplevel-list-v1.hpp
[27/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/pointer-gestures-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/pointer-gestures-unstable-v1.hpp
[28/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/keyboard-shortcuts-inhibit-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/keyboard-shortcuts-inhibit-unstable-v1.hpp
[29/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/text-input-unstable-v3.cpp, /tmp/hyprpm/hyprland-shelby/protocols/text-input-unstable-v3.hpp
[30/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/pointer-constraints-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/pointer-constraints-unstable-v1.hpp
[31/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/xdg-activation-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/xdg-activation-v1.hpp
[32/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/ext-idle-notify-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/ext-idle-notify-v1.hpp
[33/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/ext-session-lock-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/ext-session-lock-v1.hpp
[34/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/tablet-v2.cpp, /tmp/hyprpm/hyprland-shelby/protocols/tablet-v2.hpp
[35/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/presentation-time.cpp, /tmp/hyprpm/hyprland-shelby/protocols/presentation-time.hpp
[36/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/primary-selection-unstable-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/primary-selection-unstable-v1.hpp
[37/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/xdg-shell.cpp, /tmp/hyprpm/hyprland-shelby/protocols/xdg-shell.hpp
[38/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/xwayland-shell-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/xwayland-shell-v1.hpp
[39/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/viewporter.cpp, /tmp/hyprpm/hyprland-shelby/protocols/viewporter.hpp
[40/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/linux-dmabuf-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/linux-dmabuf-v1.hpp
[41/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/xdg-dialog-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/xdg-dialog-v1.hpp
[42/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/drm-lease-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/drm-lease-v1.hpp
[43/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/linux-drm-syncobj-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/linux-drm-syncobj-v1.hpp
[44/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/single-pixel-buffer-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/single-pixel-buffer-v1.hpp
[45/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/security-context-v1.cpp, /tmp/hyprpm/hyprland-shelby/protocols/security-context-v1.hpp
[46/47] Generating /tmp/hyprpm/hyprland-shelby/protocols/wayland.cpp, /tmp/hyprpm/hyprland-shelby/protocols/wayland.hpp
find src -name '*.h*' -print0 | cpio --quiet -0dump /home/shelby/.local/share/hyprpm/headersRoot/include/hyprland
cp ./protocols/*.h* /home/shelby/.local/share/hyprpm/headersRoot/include/hyprland/protocols
cp ./build/hyprland.pc /home/shelby/.local/share/hyprpm/headersRoot/share/pkgconfig
if [ -d /usr/share/pkgconfig ]; then cp ./build/hyprland.pc /usr/share/pkgconfig 2>/dev/null || true; fi
chmod -R 755 /home/shelby/.local/share/hyprpm/headersRoot/include/hyprland
chmod 755 /home/shelby/.local/share/hyprpm/headersRoot/share/pkgconfig

✔ installed headers                                                                                                                                                                                            
 ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━  5 / 5  Done!                                                                                                                                              

→ checking for updates for hyprgrass                                                                                                                                                                           
→ Cloning https://github.com/horriblename/hyprgrass                                                                                                                                                            
✔ repository hyprgrass has updates.                                                                                                                                                                            
→ Building hyprgrass                                                                                                                                                                                           
✔ found hyprpm manifest                                                                                                                                                                                        
→ Manifest has 12 pins, checking                                                                                                                                                                               
✔ commit pin 47f6fea6d297dbee8a9c5dea905783bde506fe79 matched hl, resetting                                                                                                                                    
→ Building hyprgrass                                                                                                                                                                                           
 ━━━━━━━━━━━━━━━━━━━━━━━━━╍━━━━━━━━━━━━━━━━━━━━━━━━  2 / 4  Updating hyprgrass[v] shell returned:  -> cd /tmp/hyprpm/shelby && PKG_CONFIG_PATH="/home/shelby/.local/share/hyprpm/headersRoot/share/pkgconfig" meson setup build
The Meson build system
Version: 1.6.0
Source dir: /tmp/hyprpm/shelby
Build dir: /tmp/hyprpm/shelby/build
Build type: native build
Project name: hyprgrass
Project version: v0.8.2
C compiler for the host machine: cc (gcc 14.2.1 "cc (GCC) 14.2.1 20240910")
C linker for the host machine: cc ld.bfd 2.43.0
C++ compiler for the host machine: c++ (gcc 14.2.1 "c++ (GCC) 14.2.1 20240910")
C++ linker for the host machine: c++ ld.bfd 2.43.0
Host machine cpu family: x86_64
Host machine cpu: x86_64
Compiler for C++ supports arguments -std=c++23: YES 
meson.build:8: WARNING: Consider using the built-in option for language standard version instead of using "-std=c++23".
Found pkg-config: YES (/usr/bin/pkg-config) 2.1.1
Run-time dependency hyprland found: YES 0.45.2
Found CMake: /usr/bin/cmake (3.31.0)
Run-time dependency wftouch found: NO (tried pkgconfig and cmake)

Executing subproject wf-touch 

wf-touch| Project name: wf-touch
wf-touch| Project version: 0.0
wf-touch| C++ compiler for the host machine: c++ (gcc 14.2.1 "c++ (GCC) 14.2.1 20240910")
wf-touch| C++ linker for the host machine: c++ ld.bfd 2.43.0
wf-touch| Run-time dependency glm found: YES 1.0.1
wf-touch| Run-time dependency doctest found: NO (tried pkgconfig and cmake)
wf-touch| Build targets in project: 1
wf-touch| Subproject wf-touch finished.

Run-time dependency doctest found: NO (tried pkgconfig and cmake)
Configuring version.hpp using configuration
Run-time dependency pixman-1 found: YES 0.44.0
Run-time dependency libinput found: YES 1.27.0
Run-time dependency wayland-server found: YES 1.23.1
Run-time dependency xkbcommon found: YES 1.7.0
Run-time dependency libdrm found: YES 2.4.123
Build targets in project: 3

hyprgrass v0.8.2

  Subprojects
    wf-touch: YES

Found ninja-1.12.1 at /usr/bin/ninja
                                                
 -> cd /tmp/hyprpm/shelby && PKG_CONFIG_PATH="/home/shelby/.local/share/hyprpm/headersRoot/share/pkgconfig" ninja -C build
ninja: Entering directory `build'
[1/14] Compiling C++ object src/gestures/libgestures.a.p/Shared.cpp.o
[2/14] Compiling C++ object src/gestures/libgestures.a.p/CompletedGesture.cpp.o
[3/14] Compiling C++ object src/gestures/libgestures.a.p/DragGesture.cpp.o
[4/14] Compiling C++ object subprojects/wf-touch/libwftouch.a.p/src_math.cpp.o
[5/14] Compiling C++ object subprojects/wf-touch/libwftouch.a.p/src_touch.cpp.o
[6/14] Compiling C++ object subprojects/wf-touch/libwftouch.a.p/src_actions.cpp.o
[7/14] Linking static target subprojects/wf-touch/libwftouch.a
[8/14] Compiling C++ object src/gestures/libgestures.a.p/Actions.cpp.o
[9/14] Compiling C++ object src/libhyprgrass.so.p/VecSet.cpp.o
../src/VecSet.cpp: In instantiation of ‘bool VecSet<T>::remove(T) [with T = Hyprutils::Memory::CWeakPointer<CWLTouchResource>]’:
../src/VecSet.hpp:27:16:   required from here
   27 | template class VecSet<Hyprutils::Memory::CWeakPointer<CWLTouchResource>>;
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../src/VecSet.cpp:23:23: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<Hyprutils::Memory::CWeakPointer<CWLTouchResource> >::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
   23 |     for (int i = 0; i < this->set.size(); i++) {
      |                     ~~^~~~~~~~~~~~~~~~~~
../src/VecSet.cpp:25:19: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<Hyprutils::Memory::CWeakPointer<CWLTouchResource> >::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
   25 |             if (i != this->set.size() - 1) {
      |                 ~~^~~~~~~~~~~~~~~~~~~~~~~
[10/14] Compiling C++ object src/gestures/libgestures.a.p/Gestures.cpp.o
[11/14] Linking static target src/gestures/libgestures.a
[12/14] Compiling C++ object src/libhyprgrass.so.p/GestureManager.cpp.o
FAILED: src/libhyprgrass.so.p/GestureManager.cpp.o 
c++ -Isrc/libhyprgrass.so.p -Isrc -I../src -Isubprojects/wf-touch -I../subprojects/wf-touch -I/usr/include -I/usr/include/pixman-1 -I/usr/include/libdrm -I/home/shelby/.local/share/hyprpm/headersRoot/include -I/home/shelby/.local/share/hyprpm/headersRoot/include/hyprland/protocols -I/home/shelby/.local/share/hyprpm/headersRoot/include/hyprland -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -O3 -std=c++23 -fPIC -DWLR_USE_UNSTABLE -MD -MQ src/libhyprgrass.so.p/GestureManager.cpp.o -MF src/libhyprgrass.so.p/GestureManager.cpp.o.d -o src/libhyprgrass.so.p/GestureManager.cpp.o -c ../src/GestureManager.cpp
../src/GestureManager.cpp: In member function ‘bool GestureManager::handleGestureBind(std::string, bool)’:
../src/GestureManager.cpp:204:76: error: ‘class CKeybindManager’ has no member named ‘m_vKeybinds’; did you mean ‘m_lKeybinds’?
  204 |     auto allBinds = std::ranges::views::join(std::array{g_pKeybindManager->m_vKeybinds, this->internalBinds});
      |                                                                            ^~~~~~~~~~~
      |                                                                            m_lKeybinds
[13/14] Compiling C++ object src/libhyprgrass.so.p/main.cpp.o
ninja: build stopped: subcommand failed.



✖ Plugin hyprgrass failed to build.
  This likely means that the plugin is either outdated, not yet available for your version, or broken.

```

 - OS: Arch
 - output of `hyprctl version` or `Hyprland -v`:
   
   ```
   Hyprland 0.45.2 built from branch  at commit 12f9a0d0b93f691d4d9923716557154d74777b0a  ([gha] Nix: update inputs).
   Date: Tue Nov 19 21:47:18 2024
   Tag: v0.45.2, commits: 5451
   built against aquamarine 0.5.0


   flags set:
   debug
   ```

- output of `hyprctl plugin list`:
   ```
   no plugins loaded
   ```

**Comment by matt1432** [comment]
same as https://github.com/horriblename/hyprgrass/issues/179

either go back to this commit for hyprgrass cb929099477407116031010905ce439db771dd62

or update Hyprland to latest git

**Comment by horriblename** [comment]
hyprpm pin on v0.45.2 was wrong, fixed in #181
